cartography/intel/aws/ec2/network_interfaces.py

```diff
@@ -27,256 +41,215 @@ def get_network_interface_data(boto3_session: boto3.session.Session, region: str
-            SET r.lastupdated = $update_tag
-        WITH network_interface, netinf
-        MERGE (acc:AWSAccount{id: $aws_account_id})
-        ON CREATE SET acc.firstseen = timestamp(), acc.inscope=true
-        SET acc.lastupdated = $update_tag
-        MERGE (acc)-[r:RESOURCE]->(netinf)
-        ON CREATE SET r.firstseen = timestamp()
-        SET r.lastupdated = $update_tag
-        WITH network_interface, netinf
-
-        MERGE (snet:EC2Subnet{subnetid: network_interface.SubnetId})
-        ON CREATE SET snet.firstseen = timestamp()
-        SET snet.lastupdated = $update_tag
-        MERGE (netinf)-[r:PART_OF_SUBNET]->(snet)
-        ON CREATE SET r.firstseen = timestamp()
-        SET r.lastupdated = $update_tag
-    """
-    neo4j_session.run(
-        ingest_network_interfaces, network_interfaces=data, update_tag=update_tag,
-        region=region, aws_account_id=aws_account_id,
+    for network_interface in data_list:
+        # Parse network interface description for ELB association
+        # https://aws.amazon.com/premiumsupport/knowledge-center/elb-find-load-balancer-IP/
+        elb_v1_id = None
+        elb_v2_id = None
+        elb_match = re.match(r'^ELB (?:net|app)/([^\/]+)\/(.*)', network_interface.get('Description', ''))
+        if elb_match:
+            elb_v1_id = f'{elb_match[1]}-{elb_match[2]}.elb.{region}.amazonaws.com',
+        else:
+            elb_match = re.match(r'^ELB (.*)', network_interface.get('Description', ''))
+            if elb_match:
+                elb_v2_id = elb_match[1]
+        # TODO issue #1024 change this to arn when ready
+        network_interface_id = network_interface['NetworkInterfaceId']
+        network_interface_list.append(
+            {
+                'Id': network_interface_id,
+                'NetworkInterfaceId': network_interface['NetworkInterfaceId'],
+                'Description': network_interface['Description'],
+                'InstanceId': network_interface.get('Attachment', {}).get('InstanceId'),
+                'InterfaceType': network_interface['InterfaceType'],
+                'MacAddress': network_interface['MacAddress'],
+                'PrivateDnsName': network_interface['PrivateDnsName'],
+                'PrivateIpAddress': network_interface['PrivateIpAddress'],
+                'PublicIp': network_interface.get('Association', {}).get('PublicIp'),
+                'RequesterId': network_interface.get('RequesterId'),
+                'RequesterManaged': network_interface['RequesterManaged'],
+                'SourceDestCheck': network_interface['SourceDestCheck'],
+                'Status': network_interface['Status'],
+                'SubnetId': network_interface['SubnetId'],
+                'ElbV1Id': elb_v1_id,
+                'ElbV2Id': elb_v2_id,
+            },
+        )
+        if network_interface.get('PrivateIpAddresses'):
+            for private_ip_address in network_interface['PrivateIpAddresses']:
```

Comment:
Non blocker: chane this to 
```suggestion
       for private_ip_address in network_interface.get('PrivateIpAddresses', []):
```